src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -194,7 +194,7 @@ class LedgerMaster : public AbstractFetchPackContainer
     getLedgerBySeq(std::uint32_t index);
 
     std::shared_ptr<Ledger const>
-    getLedgerByHash(uint256 const& hash);
+    getLedgerByHash(uint256 const& hash, bool acquire = true);
```

Comment:
The implications of this method used to be pretty obvious.  Now, with the added `bool acquire` argument, it's not so obvious.  It would be good to add doxygen here to document the meaning of the `acquire` argument.

Reply:
I don't agree that the implications used to be obvious if the added parameter whose default value preserves existing behavior does not have obvious implications, but I agree that more documentation will be helpful.